client/src/catalog-list.html

```diff
@@ -68,15 +68,27 @@
       url="[[baseUrls.api]]/api/search/[[query]] kind:collection[[_params(noscore)]]"
       handle-as="json"
       on-response="_onCollectionsResponse"
-      on-error="_updateLoading"></iron-ajax>
+      on-error="_updateLoading"
+      params="[[_collectionRequestParams]]"></iron-ajax>
 
     <template is="dom-if" if="[[_collections.count]]">
       <h1 class="section-header">[[_collections.count]] Collection[[_s(_collections.count)]]</h1>
       <div id="collections" class="loader" loading$="[[!_collections]]">
-        <template is="dom-repeat" items="[[_collections.results]]">
+        <!-- Don't autoload more collections if there are elements -->
+        <template is="dom-if" if="[[_elements.count]]">
+          <template is="dom-repeat" items="[[_collections.results]]">
+            <collection-card class="box" data="[[item]]"></collection-card>
+          </template>
+          <div id="spacer"></div>
+        </template>
+        <!-- But if there aren't, go for it, it's the 90s -->
+        <template is="dom-if" if="[[!_elements.count]]">
+          <iron-scroll-threshold on-lower-threshold="_loadMoreCollections" id="threshold" scroll-target="document" lower-threshold="800">
```

Comment:
hmm, I think we should move the logic to a _loadMore function instead, it would decide which one to extend. we probably don't need to change the template at all then.